### ISSUE TITLE
Can't determine actual step position to apply logic...

### ISSUE THREAD

**Issue by xomnot** [stepstone-tech/android-material-stepper#50]
I'm actually becoming crazy with this library... 

I'm trying to implement it on my app. I decided to use the Styled Tabs version. I have the three fragments you provide on sample edited for the use I need, so on the main one I show a list,  a Spinner on the second and another list on the third one.

But as I've read on other issue, 

> By default, ViewPager if you're on page N will create page N - 1 and N + 1. So if you're starting with the first page it will create the first and the second Fragment.

So, when I try to populate the list on the first fragment, or the spinner on the second, as it doesn't exists on the others, it gives me a null exception... I'm doing this on StyledTabsActivity because here it is where I found at the example that you apply the logics to the button that appears, but you have the same button with same id on the three fragments...

How or where I am supposed to be able to populate this elements?

**Comment by zawadz88** [comment]
Hi @xomnot,
I don't quite understand:

> So, when I try to populate the list on the first fragment, or the spinner on the second, as it doesn't exists on the others, it gives me a null exception... I'm doing this on StyledTabsActivity because here it is where I found at the example that you apply the logics to the button that appears, but you have the same button with same id on the three fragments...

Could you please explain what is throwing the NPE? Could you share the code that's not working?
Also, are you perhaps trying to reference Views created in Fragments from the Activity?

**Comment by xomnot** [comment]
I'm referencing the views from here [https://github.com/stepstone-tech/android-material-stepper/blob/badcd78e5373df231b0c92ebd140ae2930678e3b/sample/src/main/java/com/stepstone/stepper/sample/step/fragment/StepFragmentSample.java](StepFragmentSample) as I saw on the sample that you referenced the button.

The NPE is being thrown when the view is not visible at the actual step... I don't know if I explain myself.
Maybe I'm referencing from the wrong site. 

Right now I've solved it checking if the view is null 
`if (null != listview) {... here my code...}`
